code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/MetaDataInfoDaoImpl.java

```diff
@@ -94,7 +94,8 @@ protected ContainerMetaData map(List<Object> input) {
                 .and(instance.REMOVED.isNull())
                 .and(instance.STATE.notIn(CommonStatesConstants.REMOVING, CommonStatesConstants.REMOVED))
                 .and(exposeMap.REMOVED.isNull())
-                .and(exposeMap.STATE.notIn(CommonStatesConstants.REMOVING, CommonStatesConstants.REMOVED))
+                .and(exposeMap.STATE.isNull().or(
```

Comment:
when exposeMap entry is null. I do left outer join with this table to get service and stand alone containers